<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatronWindow</class>
 <widget class="QMainWindow" name="PatronWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patron Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="browseGroup">
      <property name="title">
       <string>Browse Catalogue</string>
      </property>
      <layout class="QVBoxLayout" name="browseLayout">
       <item>
        <widget class="QTableView" name="browseTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="browseButtons">
         <item>
          <spacer name="spacerBrowse">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnBorrow">
           <property name="text">
            <string>Borrow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPlaceHold">
           <property name="text">
            <string>Place Hold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRefreshBrowse">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="accountGroup">
      <property name="title">
       <string>My Account</string>
      </property>
      <layout class="QVBoxLayout" name="accountLayout">
       <item>
        <widget class="QLabel" name="lblLoans">
         <property name="text">
          <string>Active Loans</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="loansTable"/>
       </item>
       <item>
        <widget class="QLabel" name="lblHolds">
         <property name="text">
          <string>Active Holds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="holdsTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="accountButtons">
         <item>
          <spacer name="spacerAccount">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnReturn">
           <property name="text">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancelHold">
           <property name="text">
            <string>Cancel Hold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRefreshAccount">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
